locales/en/common.json

```
@@ -5,42 +5,42 @@
 
   "MMASK_LEAD_HEADER": "Install MetaMask",
   "MMASK_LEAD_EXPLANATION": "To use this web app, your browser needs to have MetaMask.",
-  "MMASK_MODAL_MOBILE_BODY": "Open or Install the MetaMask App",
-  "MMASK_MODAL_MOBILE_CTA": "Open the MetaMask App",
+  "MMASK_MODAL_MOBILE_BODY": "Open or Install the MetaMask app.",
+  "MMASK_MODAL_MOBILE_CTA": "Open the MetaMask app.",
   "MMASK_MODAL_BODY": "Install the browser extension for MetaMask",
   "MMASK_MODAL_CTA": "Install MetaMask",
 
   "MINT_YOUR_KAMON_HEADING": "Mint your Kamon",
-  "MINT_YOUR_KAMON_EXPLANATION": "The Kamon NFT proves your membership of the HENKAKU community.",
-  "MINT_YOUR_KAMON_DETAILS": "You can mint your Kamon NFT for 1000 $HENKAKU.",
-  "MINT_YOUR_KAMON_DETAILS_NOTE": "(Plus gas)",
+  "MINT_YOUR_KAMON_EXPLANATION": "The Kamon NFT proves that you are a member of the Henkaku community.",
+  "MINT_YOUR_KAMON_DETAILS": "You can mint a personalized Kamon NFT for 1000 $HENKAKU.",
+  "MINT_YOUR_KAMON_DETAILS_NOTE": "(Plus Gas Fees)",
   "MINT_YOUR_KAMON_BUTTON": "Go and Mint Your NFT",
   "MINT_YOUR_KAMON_WALLET_INSTRUCTION": "To mint your Kamon NFT, connect your wallet.",
   "MINT_YOUR_KAMON_MINTED": "🎉 Your NFT was minted! 🎉",
   "MINT_YOUR_KAMON_OPENSEA_INSTRUCTION": "See it on OpenSea",
   "MINT_YOUR_KAMON_OPENSEA_INSTRUCTION_2": "It may take some time for the NFT to appear.",
   "MINT_YOUR_KAMON_MINT_BUTTON": "Mint for 1000 $HENKAKU",
-  "MINT_YOUR_KAMON_DISABLED_BUTTON_HOLDER": "You already hold a Kamon NFT",
+  "MINT_YOUR_KAMON_DISABLED_BUTTON_HOLDER": "You already own a Kamon NFT",
   "CURRENT_HOLDERS_1": "🎉 ",
   "CURRENT_HOLDERS_2": " members have minted so far 🎉",
 
-  "ENABLE_EXPLANATION": "Enable Kamon NFT to access your $HENKAKU, in order to be able to mint.",
-  "ENABLE_BUTTON": "Enable $HENKAKU payments",
+  "ENABLE_EXPLANATION": "Please enable the website to access your $HENKAKU balance",
```

Suggestion: keep the period after the phrase, since this is not a heading or a button.

```suggestion
  "ENABLE_EXPLANATION": "Please enable the website to access your $HENKAKU balance.",
```